nixos/modules/services/x11/xserver.nix

```diff
@@ -248,11 +248,10 @@ in
 
       videoDrivers = mkOption {
         type = types.listOf types.str;
-        # !!! We'd like "nv" here, but it segfaults the X server.
-        default = [ "radeon" "cirrus" "vesa" "modesetting" ];
+        default = [ "amdgpu" "modesetting" ];
```

Comment:
Wouldn't amdgpu be selected by modesetting automatically? Doesn't hurt to explicitly prioritise it I guess.

Reply:
xf86-video-amdgpu is a different DDX than modesetting so how would it be selected without it being installed?